src/SablierMerkleLL.sol

```diff
@@ -23,17 +23,17 @@ contract SablierMerkleLL is
                                   STATE VARIABLES
     //////////////////////////////////////////////////////////////////////////*/
 
-    /// @inheritdoc ISablierMerkleLL
-    bool public immutable override CANCELABLE;
-
     /// @inheritdoc ISablierMerkleLL
     ISablierLockup public immutable override LOCKUP;
 
     /// @inheritdoc ISablierMerkleLL
-    bool public immutable override TRANSFERABLE;
+    bool public immutable override STREAM_CANCELABLE;
 
     /// @inheritdoc ISablierMerkleLL
-    MerkleLL.Schedule public override schedule;
+    bool public immutable override STREAM_TRANSFERABLE;
+
+    /// @dev See the documentation in {ISablierMerkleLL.getSchedule}.
+    MerkleLL.Schedule internal _schedule;
```

Comment:
We can also mark it as `private`. A `private` visibility goes well with a variable prefixed with `_`. It will also signal that it should not be inherited by any other contract.

Reply:
Makes sense

Reply:
https://github.com/sablier-labs/airdrops/pull/17/commits/a39cdd3e699e968003eecfb2f9233d87033eb332